lib/vsc/install/ci.py

```diff
@@ -212,27 +212,36 @@ def indent(line, level=1):
     else:
         install_cmd = "pip install"
 
+    prefix = os.path.join('$PWD', '.vsc-tox')
+
     if vsc_ci_cfg[PIP_INSTALL_TOX]:
-        pip_args += '--ignore-installed --user'
+        pip_args += '--ignore-installed --prefix %s' % prefix
+
+        # tox requires zipp: require zipp < 3.0 since newer version are Python 3 only
+        tox = '"zipp<3.0" tox'
 
         test_cmds.extend([
             install_cmd + ' --user --upgrade pip',
             # make sure correct 'pip' installation is used
-            'export PATH=$HOME/.local/bin:$PATH && %s %s tox' % (install_cmd, pip_args),
+            'export PATH=$HOME/.local/bin:$PATH && %s %s %s' % (install_cmd, pip_args, tox),
         ])
 
     elif vsc_ci_cfg[PIP3_INSTALL_TOX]:
         install_cmd = install_cmd.replace('pip ', 'pip3 ')
-        pip_args += '--ignore-installed --user'
+        pip_args += '--ignore-installed --prefix %s' % prefix
         test_cmds.append('%s %s tox' % (install_cmd, pip_args))
 
     else:
         install_cmd = install_cmd.replace('pip install', 'python -m easy_install')
         easy_install_args += '-U --user'
         test_cmds.append('%s %s tox' % (install_cmd, easy_install_args))
 
-    # make sure 'tox' command installed with --user is available via $PATH/$PYTHONPATH
-    test_cmds.append('export PATH=$HOME/.local/bin:$PATH && tox -v -c %s' % TOX_INI)
+    test_cmds.extend([
+        # make sure 'tox' command installed is available by updating $PATH
+        'export PATH=%s:$PATH && tox -v -c %s' % (os.path.join(prefix, 'bin'), TOX_INI),
+        # clean up tox installation
+        'rm -r $PWD/.vsc-tox',
```

Comment:
shouldn't use let `git clean` deal with that?

Reply:
Maybe, but we know what we need to remove, so might as well keep it specific?

Reply:
You're not sure that nothings else lingers around? Why not be thorough and let `git clean` do everything?

Reply:
We run `git clean -fxd` on Jenkins before running the tests, see `Jenkinsfile`.

This cleanup is probably not even needed (since if the tests are run again, `git clean` should clean it up), but I feel it's cleaner to clean up something temporary rather than leaving it linger.